### PR TITLE
added documentation on the ball launcher

### PR DESCRIPTION
**Pull request by vincentberenz**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

Added documentation on how to use the ball launcher.

## How I Tested

- what is written in the doc: by running it with success from pamy.
- the doc itself: compiled the html locally

### CODE REVIEW

File: C7_ball_launcher.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+-----------------------------------
+| Welcome to ball launcher beepy  |
+-----------------------------------
+
+to ssh to me: user pi, password: ---
+
+- sourcing colcon workspace
+- killing already running server
+- starting the ball launcher server on IP 10.42.31.174 and PORT 5555
+sleep() takes 1 positional argument but 2 were given
```
Comment (luator) [inline C7_ball_launcher.md:21]: Should this really be displayed? :)

**Review by luator — APPROVED** [review]

File: C7_ball_launcher.md (hunk 1 above)
Comment (vincentberenz) [reply C7_ball_launcher.md:21]: indeed ! I updated to hide this.

Comment (vincentberenz) [reply C7_ball_launcher.md:21]: oh, now not sure if you meant in the doc or on the launcher screen.
Agree it should the IP should not be the on the doc, so it now hidden.
For the launcher screen, I think it is very convenient to get the password and other information easily. The set of people having credentials to get into the lab seems low enough.

Comment (luator) [reply C7_ball_launcher.md:21]: I actually only meant the "sleep() takes 1 positional argument but 2 were given" line, this looks like it is an error in the launcher screen software?

Comment (vincentberenz) [reply C7_ball_launcher.md:21]: Actually this is an error in the launcher server. Looks like it work anyway.

**Pull request merged**